lms/tasks/annotations.py

```diff
@@ -63,6 +63,10 @@ def annotation_event(*, event) -> None:
     annotation_event = AnnotationEvent(**event)
     annotation = annotation_event.annotation
 
+    if annotation_event.action not in {"create", "update"}:
```

Comment:
This was missing here. We can rely on what H is publishing (only this type of events) but I reckon is better to be explicit to avoid problems in the future.